django_jinja/views/__init__.py

```diff
@@ -46,22 +47,24 @@ def put(self, request, *args, **kwargs):
     def patch(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
+    match_extension = get_match_extension() or ".jinja"
```

Comment:
Sorry for the late reply. I can move it to something that initializes via the AppConfig perhaps. Any recommendations?